src/poetry/core/json/schemas/poetry-schema.json

```
@@ -651,6 +651,10 @@
         "links": {
           "type": "boolean",
           "description": "Declare this as a link source. Links at uri/path can point to sdist or bdist archives."
+        },
+        "indexed": {
+          "type": "boolean",
+          "description": "For simple/PEP503 repositories, pre-fetch and index the available packages."
```

```suggestion
          "description": "For PEP 503 simple API repositories, pre-fetch and index the available packages. (experimental)"
```

Let's go with this for now, it would be also great to move validation of non-core specific configuration section into poetry with a separate schema later. So we can do these a bit easier in the future.